lib/geoengineer/cli/geo_cli.rb

```diff
@@ -137,11 +137,12 @@ def print_validation_errors(errs)
     puts "Total Errors #{errs.length}"
   end
 
-  def shell_exec(cmd, verbose = @verbose)
+  def shell_exec(cmd, verbose = @verbose, verbose_state_output = @verbose_state_output)
     stdin, stdout_and_stderr, wait_thr = Open3.popen2e({}, *cmd)
 
     puts(">> #{cmd}\n") if verbose
     stdout_and_stderr.each do |line|
+      next if line =~ /Refreshing state.../ && verbose_state_output
```

Comment:
Shouldn't this be `&& !verbose_state_output`?

For this line to be show, a user would have to set both `--verbose` and `--show-refresh-state`. 

This block might be a little easier to follow if written as follow:
```
  next unless verbose
  next if line =~ /Refreshing state.../ && !verbose_state_output
  puts(line)
```

Reply:
Yup - good call on inverted logic!
Great idea on simplifying.
Fixed!